packages/core/src/client.ts

```diff
@@ -2248,6 +2260,29 @@ export class MedplumClient extends EventTarget {
       }
     }
 
+    if (response.status === 202) {
```

Comment:
Another consideration is to move the content location status polling to [bulkData](https://github.com/medplum/medplum/pull/2071/files#diff-858339789c798051d9c13620bffce8ce860e7ad8ba18dba1c8711eb1f7d51a55R2161)